AnkiConnect.py

```diff
@@ -1110,11 +1118,14 @@ def addNote(self, note):
     def addNotes(self, notes):
         results = []
         for note in notes:
-            params = AnkiNoteParams(note)
-            if params.validate():
-                results.append(self.anki.addNote(params))
-            else:
-                results.append(None)
+            try:
+                params = AnkiNoteParams(note)
+                if params.validate():
+                    results.append(self.anki.addNote(params))
+                else:
+                    results.append(None)
+            except Exception as e:
+                results.append(str(e))
```

Comment:
This change would mean that `addNotes` can return a note ID, None or a string. Existing code in things that use AnkiConnect likely have code that checks to see if result `is not None`, and this would break their behavior.

Reply:
Changed to append(None) instead